extensions/unix-timestamp/src/timestamp-to-date.tsx

```diff
@@ -0,0 +1,44 @@
+import { showToast, ActionPanel, Icon, SubmitFormAction, Form, ToastStyle } from '@raycast/api';
+import { getCurrentTimestamp, getDate, getRelativeTime, toDateString } from './utils';
+
+interface Form {
+  timestamp: string;
+}
+
+export default function main() {
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <ConvertAction />
+        </ActionPanel>
+      }
+    >
+      <Form.TextField id="timestamp" title="Timestamp" />
+    </Form>
+  );
+}
+
+function ConvertAction() {
+  async function handleSubmit(values: Form) {
+    const { timestamp } = values;
+    if (timestamp.length === 0) {
+      showToast(ToastStyle.Failure, 'Empty input');
```

Comment:
Looks like a `return` statement is missing here?

Reply:
Right, fixed